src/components/Round.tsx

```diff
@@ -164,7 +165,10 @@ export default function Round(props: Props) {
                 }
                 props.dispatch({
                   type: 'NEXT_ROUND',
-                  payload: closedRound,
+                  payload: {
+                    closedRound,
+                    gremlin: props.rollGremlin(),
```

Comment:
the gremlin for next round will be rolled for when previous round is being closed. But it's then stored in the new round.